pyfr/ctypesutil.py

```diff
@@ -11,8 +11,8 @@ class LibWrapper:
     _statuses = None
     _functions = None
 
-    def __init__(self):
-        self._lib = lib = load_library(self._libname)
+    def __init__(self, mode=ctypes.DEFAULT_MODE):
```

Comment:
Probably easier to use `_mode = ctypes.DEFAULT_MODE` as a member variable which sub classes can override directly.  Also break this change into its own PR.